src/openms/source/FORMAT/OSWFile.cpp

```diff
@@ -304,7 +304,7 @@ namespace OpenMS
       {
         int id = Sql::extractInt(stmt, I_PROTID);
         accession = Sql::extractString(stmt, I_ACCESSION);
-        swath_result.addProtein(OSWProtein(std::move(accession), id, {}));
+        swath_result.addProtein(OSWProtein(accession, id, {}));
```

Comment:
already temporary

Reply:
?? not really, the two lines are clearly different. The former could use a hypothetical move constructor `OSWProtein(String&&, int)` while the latter cannot. One would have to write one though

Reply:
ah right... maybe not worth the trouble?